Sources/URLRequest+Sweetness.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+extension URLRequest {
+    public var debugDescription: String {
```

Comment:
It is, `debugDescription` is defined by `NSObject`. Anything that inherits from it has it.